common/lib/protocol-definitions/src/clients.ts

```diff
@@ -5,27 +5,77 @@
 
 import { IUser } from "./users";
 
+/**
+ * A client's connection mode. Readonly or read/write.
+ */
 export type ConnectionMode = "write" | "read";
 
+/**
+ * TODO
+ */
 export interface ICapabilities {
+	/**
+	 * Indicates if the client represents a potentially interactive session with a user (if 'true') or if it's a "system entity" (if 'false').
+	 * The only "system entity" scenario at the moment is the summarizer client.
```

Comment:
@alexvy86 is there a good API target we can link to for the `summarizer` here? If I were reading this comment for the first time, that is additional context I would find useful.

Reply:
`@fluidframework/container-runtime#Summarizer`. We won't be able to link to it directly given this is a lower-layer package, but I guess a manual pointer to the package + class name is good.

Reply:
Hmm, if this is a lower level concept, should we omit this note then? Is there a substantial risk it will become invalid in the near future? If so, I'd probably prefer to remove it.

Reply:
I don't think the situation will change anytime soon. AFAIK we don't have plans for other "system entities", and I don't see the summarizer going away.